ll.tm

```diff
@@ -4992,11 +4992,22 @@ ParamAttribute -> ParamAttribute
 	| Dereferenceable
 	| ParamAttr
 	| Preallocated
+	| StructRetAttr
+	| ByRefAttr
```

Comment:
I'll sort the non-terminal production rules alphabetically (in a follow-up commit). We can try to figure out if there is a way to automate this in GoLand.